apps/website/src/app/atoms/icons/GithubLogo.tsx

```diff
@@ -1,9 +1,9 @@
-export type IconProps = {
+export type GithubLogoProps = {
   svgStyle: string;
   fillColor?: string;
 };
 
-export const GithubLogo = (props: IconProps) => {
+export const GithubLogo = (props: GithubLogoProps) => {
```

Comment:
Is this still required?

Reply:
No, removed the specific props and replaced them with SVGProps. Additionally, I removed the `GithubHoverableLogo` and adjusted `GithubLogo` to fit both purposes.